P6-Setup-App-Structure/content.md

```diff
@@ -65,25 +70,25 @@ Great! When you run the app now, you should see a tab bar with two items:
 
 #Adding a Third View Controller to the Tab Bar
 
-The default Tab Bar comes with two View Controllers. Our outline shows that we need three View Controllers in the Tab Bar for *Makestagram*. Let's add that third View Controller.
+The default tab bar comes with two view controllers; however, we need three view controllers for **Makestagram**. Lets add the third view controller now.
 
 > [action]
-Add a third View Controller, as shown in the video below. Note that the key you need to hold down is the *control* key which has the following symbol: ⌃
+Add a third view controller, as shown in the video below. Note that the key you need to hold down is the *control* key which has the following symbol: ⌃
 <video width="100%" controls>
   <source src="https://s3.amazonaws.com/mgwu-misc/SA2015/AddViewController_TabBar_Small.mov" type="video/mp4">
 </video>
 
-Now we have all required View Controllers set up.
+We now have all of the required view controllers set up!
 
 #Decorating the Tab Bar Items
 
-The empty Tab Bar looks pretty sad - let's add an image to each Tab Bar Item. That will also make it easier to identify the different tabs.
-
-Currently the Tab Bar has three entries and looks like this:
+Currently the tab bar has three entries and looks like this:
 
 ![image](tab_bar.png)
 
-Each of these three entries is called a *Tab Bar Item*. Every View Controller that has a relationship to a Tab Bar View Controller owns one such a *Tab Bar Item*. The Tab Bar Items are named rather generically right now (*Item 1*, *Item 2*, etc.). Instead of names, we will use icons to decorate our *Tab Bar Items*.
+<!-- Every View Controller that has a relationship to a Tab Bar View Controller owns one such a *Tab Bar Item*. The Tab Bar Items are named rather generically right now (*Item 1*, *Item 2*, etc.). Instead of names, we will use icons to decorate our *Tab Bar Items*. -->
+
+The empty tab bar looks pretty bad, so let's add an image to each tab bar item to make things look nicer. (Images will also have the benefit of making it easier to identify the different tabs.)
 
```

Comment:
please see line 17, as I added most of the details about tab bar and tab bar items there